### PR TITLE
[Fix] Update the style of modal

### PR DESCRIPTION
**Pull request by zack76**
1. Update the transition style of modal on mobile view
2. Update the style of close button
3. Update the style of checkbox error message

**Before**
Screen Shot 2021-11-08 at 11 39 16

**After**
Screen Shot 2021-11-08 at 11 38 59

### CODE REVIEW

File: src/components/Modal/Modal.tsx [hunk 1]
```diff
@@ -46,7 +49,7 @@ const Modal: ModalType = ({
             isActive={isVisible}
             shouldMountOnEnter={true}
             shouldUnmountOnExit={true}
-            type="zoom"
+            type={isMobile ? 'slideDown' : 'zoom'}
```
Comment (hassanraza0054) [inline src/components/Modal/Modal.tsx:52]: Assuming you got a UI review on the new animation type. Nicely done 👌

**Review by hassanraza0054 — APPROVED** [review]

**Review by cyrusxzw — APPROVED** [review]

File: src/components/Checkbox/Checkbox.module.css [hunk 2]
```diff
@@ -82,12 +82,7 @@
 
 .errorMessage {
   display: inline-block;
-  margin-left: 23px; /* box width + left margin */
   color: var(--color-error);
   font-family: var(--font-family-sans-serif-regular);
   font-size: rem(14px);
-
-  @media (--viewport-xs-sm-only) {
-    font-size: rem(12px);
```
Comment (MkMan) [inline src/components/Checkbox/Checkbox.module.css:91]: The error message will be larger than the actual checkbox text. Is that ok?

**Review by MkMan — APPROVED** [review]
All good as long as the UX team is good with these changes 😃

File: src/components/Checkbox/Checkbox.module.css (hunk 2 above)
Comment (zack76) [reply src/components/Checkbox/Checkbox.module.css:91]: Ah, it wont happen. the size of both will be 14px

File: src/components/Modal/Modal.tsx (hunk 1 above)
Comment (zack76) [reply src/components/Modal/Modal.tsx:52]: Yes, it's passed the UI review already and this change was requested by Michael actually.

File: src/components/Checkbox/Checkbox.module.css (hunk 2 above)
Comment (MkMan) [reply src/components/Checkbox/Checkbox.module.css:91]: Oh, maybe I misunderstood the code on line 79 then 🤔

Comment (zack76) [reply src/components/Checkbox/Checkbox.module.css:91]: oh damn, i forgot i had updated the font size sin web ui. no need change here. Thank you!

**Review by kevin-ho87 — APPROVED** [review]

**Pull request merged**